src/app/pages/create-account/create-account.page.ts

```diff
@@ -13,7 +13,6 @@ import { PasswordValidator } from '../../validators/password.validator';
 })
 export class CreateAccountPage implements OnInit {
 
-  firstName: '';
```

Comment:
So 'firstName' is no longer a standalone property but rather a value within the signupForm for 'currentUser', right?